### DISCUSSION TITLE
Demonstration Map Tab in WeAreMahsaAmini.com

### DISCUSSION THREAD

**Discussion by JeffShomali** [WeAreMahsaAmini/Core#37] [Ideas]
## Problem

<details>

> Display Demonstration Map By Calendar

As you know, many Telegram pages (i.e., the Kooshyar T channel) and Facebook groups share the next demonstration date and location. It's not organized, and people might be lost in their plans for the upcoming days and weeks.

</details>

## Solution

<details>
           <summary>Solution</summary>

> Create a clickable map by location to display the next demonstration date (basically event based map) 

- Collect data from users by form and display it on the map
- Upcoming and Past events
- Filter By Country 

</details>

<img width="800" alt="Screen Shot 2022-09-27 at 9 21 38 AM" src="https://user-images.githubusercontent.com/6022903/192582437-4525469e-854e-40d4-989a-e07bfa5884dd.png">

**Comment by azgi** [comment]
There is already a map available here: https://iranrally.ushahidi.io/views/map - maybe we should reach out and incorporate the page in the website if they agree.
